src/cpp/continuous_batching/CMakeLists.txt

```diff
@@ -72,7 +71,7 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(googletest)
 
-
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
I think we can set this property only once for CB library itself and remove from all other places.